receiver/jaegerreceiver/config.go

```diff
@@ -68,8 +59,7 @@ func DefaultServerConfigUDP() ServerConfigUDP {
 
 // Config defines configuration for Jaeger receiver.
 type Config struct {
-	Protocols      `mapstructure:"protocols"`
-	RemoteSampling *RemoteSamplingConfig `mapstructure:"remote_sampling"`
```

Comment:
As this is a breaking change for end users, i would recommend using a featuregate to remove, pinging code owner @jpkrohling for confirmation. See https://github.com/open-telemetry/opentelemetry-collector/blob/main/CONTRIBUTING.md#end-user-impacting-changes.

Reply:
I agree. I would make this no-nop for now, logging a WARN in case it's used. On the next version, a feature gate, followed by the effective removal.

Reply:
Hello, thank you for your comments. This is already a no-op. Is it ok if I add a feature gate which allows users to "enable" this feature in a no-op way? Therefore the default behaviour will be to return an error if the there is "remote_sampling" in the config, and suggest to them they could either enable the feature gate or change their config?

Reply:
Sounds good to me.